CogNative/main.py

```diff
@@ -32,8 +32,25 @@
     print(colorize("ERROR: Enter an output .wav file", "error"))
     exit(1)
 
+v.set_file_path(file_path)
+
+embedding = input("Use embedding? (y/n)\n")
+if embedding.lower() in ['y', 'yes']:
+    embedding_path = v.get_embedding_path()
```

Comment:
```RTVC.get_embedding_path()``` will be used to find embedding path, rather than the user entering it. This could be changed.